packages/runtime/test/transport/messages.test.ts

```diff
@@ -189,4 +190,13 @@ describe("Message query", () => {
         expect(messagesToRecipient2.value).toHaveLength(1);
         expect(messagesToRecipient1Or2.value).toHaveLength(2);
     });
+
+    test("query Messages withAttachments", async () => {
+        await exchangeMessageWithAttachment(transportServices1, transportServices2);
+        await exchangeMessage(transportServices1, transportServices2);
+
+        const messages = await transportServices2.messages.getMessages({ query: { attachments: "+" } });
+
+        expect(messages.value.every((m) => m.attachments.length > 0)).toBe(true);
+    });
```

Comment:
That test is weird. :) I would save the two messages and in the assertion, I would do the following assertions:

```c#
expect(messages).toHaveLength(1);
expect(messages[0].id).toBe(messageWithAttachments.Id)
```

Reply:
actually I can't rely on having an empty list of messages when there were other tests before.

Reply:
But I can add the second check, that the first message is contained in the array.

Reply:
Ahh, I expected the transportServices to be reinitialized before each test.

Reply:
Is it better now?